BuildForSDG/Team-037-Product-backend
# refactor: refactor code and folder structure

#### What does this PR do?
Tell us what this pull-request you are raising will add to the project.

#### Description of Task to be completed?
list the things you did to complete this project
- refactor codebase

#### How should this be manually tested?
Tell us how other people can test out this your feature to see if it's working as expected.
- Use git clone to clone this repository to the local branch
- Use npm install to download all necessary packages
- Use npm run test to test
#### Any background context you want to provide?
Anything you'll like to say about this pull-request?

#### What are the relevant pivotal tracker stories? (if applicable)
Type in the pivotal-tracker ID of the task/ticket you completed and the link to that ticket.
[](url)

#### Screenshots
take pictures of the implementation of the project and paste them here


Please clear out all the descriptions, including this one as soon as you're done.
